### DISCUSSION TITLE
O começo na profissão/Faculdade EAD

### DISCUSSION THREAD

**Discussion by tomasluna** [frontendbr/forum#1967] [Discussão]
Boa galera, sou formado em comércio exterior e trabalho na área há um bom tempo já, mas quero começar a estudar TI e trabalhar com isso no futuro, pois é algo que sempre me interessou. Eu tive apenas uma aula de web design numa faculdade de multimídia que fiz e vi um pouco com programação, fora isso nunca estudei mais nada. To pensando em começar uma faculdade a distância agora na Unopar, alguém sabe me dizer se lá é realmente bom? Possuem uma recomendação de faculdade melhor? Outras recomendações? Queria entrar o quanto antes no mercado de trabalho, não aguento mais minha área :(

Grato :)

**Comment by sarahdialmeida** [comment]
Oi Tomas, que ótimo que você quer ingressar na área de Ti. Eu comecei há um
ano atrás a transição de carreira e penso que foi das decisões mais
acertadas da vida. Creio que a ideia de fazer uma nova graduação é muito
boa pra fazer networking e talvez também pra conhecer conceitos teóricos da
área de tecnologia, mas se você quer mesmo programar, pelo que tenho
entendido, a universidade não vai te oferecer esse conhecimento, sobretudo
em se tratando de linguagens e tecnologias que pedem o mercado. Sendo o seu
objetivo ingressar o quanto antes no mercado de trabalho te recomendo
buscar um Bootcamp. Eu fiz o da Cubos Academy e recomendo muito. É
absolutamente excelente. Mas existem outros também como por exemplo
Rockseat, Origamid e vários cursos grátis. Muita sorte! :)

Em ter., 5 de jan. de 2021 às 09:05, tomasluna <notifications@github.com>
escreveu:

**Reply by tomasluna** [reply]
Oi Sarah, primeiramente obrigado pela resposta :)

Você conseguiu ingressar na área com esse curso e sem uma graduação? Pensei que cursando seria mais fácil de encontrar um estágio...

**Comment by sarahdialmeida** [comment]
Oi Tomas, eu acabei de terminar o Bootcamp e me sinto totalmente preparada
pra concorrer a uma vaga como júnior. Estou agora na fase de organizar
portfólio/ estudar para entrevistas técnicas/ me inscrever em vagas. Se
você coloca bastante energia e foco, sem dúvida alguma você consegue. Por
isso creio que o Bootcamp é excelente pra dar esse norte na construção da
carreira. Antes dele eu estudava várias coisas aleatórias e estava bem
perdida sobre como realmente entrar no mercado.

Em ter., 5 de jan. de 2021 às 09:41, tomasluna <notifications@github.com>
escreveu:

**Reply by tomasluna** [reply]
Entendo, creio que estou perdido assim como você também estava rs vou já começar a ler sobre o bootcamp.

Muito obrigado, já me ajudou bastante!
